src/state/slices/preferencesSlice/selectors.ts

```diff
@@ -2,36 +2,20 @@ import type { AssetId } from '@shapeshiftoss/caip'
 import createCachedSelector from 're-reselect'
 
 import type { Flag } from './preferencesSlice'
+import { preferences } from './preferencesSlice'
 
-import { isMobile as isMobileApp } from '@/lib/globals'
 import type { ReduxState } from '@/state/reducer'
 
-export const selectFeatureFlags = (state: ReduxState) => state.preferences.featureFlags
+export const selectSelectedLocale = preferences.selectors.selectSelectedLocale
```

Comment:
nitpick - remove and use slice.selectors

Reply:
8cd6d2ec51